common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -832,3 +832,34 @@ beerocks::eBssType wireless_utils::wsc_to_bwl_bss_type(WSC::eWscVendorExtSubelem
 
     return beerocks::BSS_TYPE_INVALID;
 }
+
+std::list<uint8_t> string_to_wsc_oper_class(const std::string &operating_class)
+{
+    std::list<uint8_t> radio_24g = {81, 82, 83, 84};
```

Comment:
```suggestion
    const std::vector<uint8_t> radio_24g {81, 82, 83, 84};
```
- [Prefer `{}` initializer](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-list)
- [SL.con.2: Prefer using STL vector by default unless you have a reason to use a different container](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#slcon2-prefer-using-stl-vector-by-default-unless-you-have-a-reason-to-use-a-different-container)
- [Con.4: Use const to define objects with values that do not change after construction](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rconst-const)

(I think in C++20 we could even use constexpr for vectors, but not in C++14)

Reply:
We can't use const because merge() method does not work for const lists.